nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/impl/ListProperty.java

```diff
@@ -243,7 +243,11 @@ public void setValue(Object value) throws PropertyException {
         if (value == null) {
             List<Property> temp = new ArrayList<Property>(children);
             for (Property p : temp) { // remove all children
-                p.remove();
+                p.remove(); // call #remove(Property) which set isModified flag
```

Comment:
i am wondering about the comment ` call #remove(Property) which set isModified flag` you want to call `remove(p);` or some thing else. and it can be replaced ` temp.forEach(this::remove);` but i think this is a cleanup purpose :) and we don't want it on backport

Reply:
This is because under the hood this will call `org.nuxeo.ecm.core.api.model.impl.AbstractProperty.remove()`
This method will call the parent of the prop (the list, and make a call to remove(Property) on line 138